package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "19.3.1",
+  "version": "19.3.2",
```

Comment:
As this is a little new feature I would go with 19.4.0

Reply:
done